bin/offline-dbg

```diff
@@ -133,6 +142,138 @@ EOF
     echo "**************************************************************"
 }
 
+get_highest_ofp_version() {
+    # if no protocol listed, default to OpenFlow14
```

Comment:
Why? Why not OpenFlow15?

Reply:
My thinking is 1.4 is enabled by default on 2.8-2.11 while 1.5 is only enabled on 2.12 on. Seemed like a safer choice. ovs-save states that only 14 is needed to restore flows.

Reply:
It feels weird to have a function called get_hightest_... that does not default to the highest. If it's only going to be used to verify it we can restore the flows at all maybe refactor the function to perform the check and rename it as "check_ofp_version" or something like that?

Reply:
Makes sense to me, will do